context.go

```diff
@@ -383,6 +383,15 @@ func (c *Context) Param(key string) string {
 	return c.Params.ByName(key)
 }
 
+// SetParam adds param to context an
+// replaces path param key with given value for e2e testing purposes
+// Example Route: "/user/:id"
+// SetParam("id", 1)
+// Result: "/user/1"
+func (c *Context) SetParam(key, value string) {
```

Comment:
maybe change name to `AddParam`

Reply:
Good suggestion, I changed the name to `AddParam`. I also ran `gofmt -s -w context.go && gofmt -s -w context_test.go` to fix https://github.com/gin-gonic/gin/runs/3468382048?check_suite_focus=true 
Please let me know if there is more feedback.